oap-server/server-fetcher-plugin/prometheus-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/fetcher/prometheus/provider/PrometheusFetcherConfig.java

```diff
@@ -28,13 +28,15 @@
 
 @Getter
 public class PrometheusFetcherConfig extends ModuleConfig {
+
+    private int meterConvertWorker = Runtime.getRuntime().availableProcessors() / 2;
```

Comment:
You will face a bug when `Runtime.getRuntime().availableProcessors()`=1, I think. 
And take a look at `BulkConsumePool`, this usually accepts a System ENV override.

Reply:
I would like to default the worker number to `Runtime.getRuntime().availableProcessors()`(the min value should be 1). An env could override it.

Reply:
> I would like to default the worker number to `Runtime.getRuntime().availableProcessors()`(the min value should be 1). An env could override it.

Really need so many threads? This should not cost too many resources, as prom data structure is not complex and data set is not very big.

Reply:
I will avoid this case, core == 1. But I don't think we need EnvUtil to do it, because we have provided a configuration option that can override the value.